www/js/socialmediashare.js

```diff
@@ -94,7 +94,7 @@ define(['jquery', 'wdn', 'require', 'wdn-ui'], function($, WDN, require, wdnUI)
       }
     },
 
-    shareButtonTemplate: '<button class="wdn-dropdown-widget-button" aria-pressed="false" aria-haspopup="true" aria-controls="wdn_share_{{id}}"><span class="wdn-icon-share" aria-hidden="true"></span><span class="wdn-text-hidden">Share This Page</span></button>'+ // add string to use as template for all share buttons
+    shareButtonTemplate: '<button class="wdn-dropdown-widget-button" aria-pressed="false" aria-haspopup="true" aria-controls="wdn_share_{{id}}"><span class="wdn-icon-share" aria-hidden="true"></span><span class="dcf-sr-only">Share This Page</span></button>'+ // add string to use as template for all share buttons
```

Comment:
This file has several wdn class references (mostly wdn-icon-*).  I was unable to verify that this file is actually needed.  If not we should just remove it.  It appears to only be used by `submission.js` for `createGoURL` functionality.